include/config/config_camera.h

```diff
@@ -52,3 +56,22 @@
 #ifdef ENABLE_VANILLA_LEVEL_SPECIFIC_CHECKS
     #define ENABLE_VANILLA_CAM_PROCESSING
 #endif
+
+// Reonucam overrides
+#ifdef REONUCAM
+    // Use course default mode
+    #ifndef USE_COURSE_DEFAULT_MODE
+    #define USE_COURSE_DEFAULT_MODE
+    #endif
+
+    // Force camera mode to 8 Dir
+    #ifdef FORCED_CAMERA_MODE
+    #undef FORCED_CAMERA_MODE
+    #endif
+    #define FORCED_CAMERA_MODE CAMERA_MODE_8_DIRECTIONS
+
+    // Disable vanilla cam processing
+    #ifdef ENABLE_VANILLA_CAM_PROCESSING
+    #undef ENABLE_VANILLA_CAM_PROCESSING
+    #endif
+#endif
```

Comment:
maybe we should put this in safeguards? idk honestly, makes sense either way

Reply:
If we ever want to make a config tool, I'd say yes since it'll make implementing parsing easier. It also adds general clutter by leaving it here.